### PR TITLE
 Completed react-toggle-switch-component

### PR DESCRIPTION
**Pull request by Alex-Davis0**
web

### CODE REVIEW

**Review by scbowler — APPROVED** [review]
Would have liked to see the background color change too, but overall good job! 👍

**Comment by Alex-Davis0** [comment]
web

File: react-toggle-switch-component/src/toggle-switch.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: !this.state.isClicked });
+  }
+
+  render() {
+    return (
+      <div className={this.state.isClicked ? 'border-gray' : 'border' }>
```
Comment (scbowler) [inline react-toggle-switch-component/src/toggle-switch.jsx:15]: Nice! 👍

**Review by scbowler — APPROVED** [review]
Great adjustment, looks good! 👍

**Pull request merged**